cs.dict.combodo-approval-extended.php

```diff
@@ -95,7 +95,7 @@
 	'Class:ApprovalRule/Attribute:level2_substitute_query+' => 'Substitutes are approver dependent : use `:approver->...` placeholder in the query to retrieve the corresponding substitutes~~',
 	'Class:ApprovalRule/Attribute:level2_substitute_timeout' => 'Substitute notification delay L2~~',
 	'Class:ApprovalRule/Attribute:level2_substitute_timeout+' => 'Substitutes will be notified if approver has not answered before this percentage of the approval delay~~',
-	'Class:ApprovalRule/Attribute:servicesubcategory_list' => 'Podkategorie služeb',
+	'Class:ApprovalRule/Attribute:servicesubcategory_list' => 'Service subcategory~~',
```

Comment:
Is setting back the english version of this key really intended ?

Reply:
This is an error from our internal merging scripts :-(
The previous translation is correct.